app/web/src/newhotness/Workspace.vue

```diff
@@ -91,10 +80,26 @@
         :loadingSteps="_.values(muspelheimStatuses)"
       />
     </Transition>
-
-    <main v-if="lobby && tokenFail" class="grow min-h-0">
-      <div>Bad Token</div>
-    </main>
+    <template v-if="!lobby">
```

Comment:
OK, so the expected behavior for how this is written is:
1. user comes to page
2. hits lobby
3. hits onboarding

Just wanted to confirm thats what we all decided on 😅

Reply:
OK, i see how the computed `lobby`changed. Now it will be:
1. lobby = false, onboarding
2. lobby = true, in lobby
3. lobby = false, out of lobby

Reply:
First comment is right user hits page, gets lobby until:cold boot is done, feature flags are loaded And on-boarding completion data is loaded. That is so we don't blink anything. Lobby blink time was also updated so we don't blink the lobby if this takes less than 2 seconds